.github/workflows/tests.yml

```diff
@@ -71,17 +71,23 @@ jobs:
       - name: Install nextest
         uses: taiki-e/install-action@nextest
       - name: Test forest
-        run: make -C forest test
+        run: |
+          cd forest
+          make test
         env:
           CC: "sccache clang"
           CXX: "sccache clang++"
       - name: Install forest
-        run: make -C forest install
+        run: |
+          cd forest
+          make install
         env:
           CC: "sccache clang"
           CXX: "sccache clang++"
       - name: Calibnet health check
-        run: ./forest/scripts/tests/calibnet_other_check.sh
+        run: |
+          cd forest
+          ./scripts/tests/calibnet_other_check.sh
```

Comment:
Why?

Reply:
@lemmih The script can only import the test snapshot properly when CWD is forest root

Reply:
Tracking issue: https://github.com/ChainSafe/fil-actor-states/issues/165